packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -91,7 +102,9 @@ class ComponentWrapper extends React.Component<IComponentWrapperProps<IDevSiteDe
     }
 
     private getClassNames(): string {
-        const classNames: string = this.props.managedClasses.componentWrapper;
+        const classNames: string = this.props.transparent
```

Comment:
with the other change to this in your PR, this will get even more complex and difficult to read. Suggest breaking this up here as well.